test/integration/acceptance/query_permission_fixture.hpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
```

Comment:
same here, check the rest by yourself please

Reply:
done. now in accordance with [google style guide](https://google.github.io/styleguide/cppguide.html#The__define_Guard)